Platforms/QemuQ35Pkg/QemuQ35Pkg.dec

```diff
@@ -71,15 +71,9 @@
   #
   gQemuQ35PkgTokenSpaceGuid = { 0xb7048716, 0x7d8b, 0x4a86, { 0x84, 0x95, 0x57, 0xa1, 0x76, 0x4f, 0x6e, 0xde } }
 
-  ## Example Profile 1 will be stored in FV under this GUID
-  gQemuQ35PkgProfile1Guid = { 0xe34d00d0, 0x6a10, 0x44be, { 0xb4, 0x6c, 0xbe, 0xe6, 0x30, 0x2c, 0x62, 0x87 } }
-  ## Example Profile 2 will be stored in FV under this GUID
-  gQemuQ35PkgProfile2Guid = { 0x848f7e93, 0xc957, 0x4797, { 0x8a, 0x11, 0xf3, 0x01, 0xed, 0x9b, 0x20, 0x48 } }
-   ## Example Profile 3 will be stored in FV under this GUID
-  gQemuQ35PkgProfile3Guid = { 0x454cfa58, 0x6423, 0x4f50, { 0x8b, 0x2b, 0x74, 0x4b, 0xde, 0xce, 0x87, 0x6a } }
-
-  ## The Platform DFCI Unsigned list file GUID
-  gQemuQ35PkgDfciUnsignedXmlGuid = { 0x62cf29ad, 0xfeee, 0x4930, { 0xb7, 0x1b, 0x48, 0x6, 0xc7, 0x87, 0xc6, 0xaa } }
+  ## QemuQ35 Platform Config Knob Namespace Guid
+  #
+  gQemuQ35PkgConfigKnobNamespaceGuid = { 0x5e6a1994, 0xe683, 0x4d8c, { 0x9d, 0xc1, 0xf1, 0xf6, 0x34, 0x90, 0xfc, 0x60 } }

```

Comment:
I might have missed (again), but where is this used?

Reply:
This is only used in the config XML as the namespace GUID. I added it here originally with the idea that it would be nice to have an official definition for it somewhere, but I don't think it matters, the XML can have whatever GUIDs it wants. I'll remove it.